style.css

```diff
@@ -0,0 +1,81 @@
+#header {
+}
```

Comment:
Unnecessary selectors, if they are not used it's better to delete them.

Reply:
I would suggest removing any unnecessary selectors that are not being used to improve the styling in that way you maintain a more organized CSS code.